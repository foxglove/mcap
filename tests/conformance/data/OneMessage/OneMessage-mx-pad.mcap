version https://git-lfs.github.com/spec/v1
oid sha256:7bd3dd3e93da135e2637d943bcd52f2b49baab86ed93a24b63e7b7863472cf0c
size 202
