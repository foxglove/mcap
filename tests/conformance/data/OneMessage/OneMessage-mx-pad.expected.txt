Header library=s:"" profile=s:""
Schema data=b:<040506> encoding=s:c id=i:1 name=s:Example
ChannelInfo id=i:1 message_encoding=s:a metadata=m:{s:foo=s:bar} schema_id=i:1 topic=s:example
Message channel_id=i:1 data=b:<010203> log_time=n:2 publish_time=n:1 sequence=i:10
Footer summary_crc=i:0 summary_offset_start=n:0 summary_start=n:0
